app/views/articles/index.html.erb

```diff
@@ -36,7 +36,8 @@
     <%= render "articles/sidebar" %>
 
     <%# BEGIN Feed menu bar %>
-    <div class="articles-list crayons-layout__content" id="articles-list">
+    <main class="articles-list crayons-layout__content" id="articles-list" role="main">
+      <h1 style="position: absolute; left: -10000px; width: 1px; height: 1px; overflow: hidden;">Articles</h1>
```

Comment:
This is something worth discussing.

We should think about doing this in more places to improve the performance of screen readers. I added this here to satisfy axe, but it could be used all over the app.

There are a few methods for adding hidden headings that will still be useful for screen readers, this is just the one I picked.